graphics/pdfpc/Portfile

```diff
@@ -39,27 +39,35 @@ checksums               rmd160  ae78dbb12ec2f639705a09ecd67055b17599ea0e \
                         sha256  04de35626c415ad4105dd9fb75e92b3f05c5e83a3728b1676ee1ef2e6581d03c \
                         size 7733169
 
+configure.args          -DMOVIES=off
+
 pre-configure {
     if {![variant_isset quartz] && ![variant_isset x11]} {
         error "Either +x11 or +quartz is required"
     }
 }
 
-variant quartz conflicts x11 {
+variant quartz conflicts x11 description {
```

Comment:
> If you add a `description` keyword, you'll [need do provide a description](https://guide.macports.org/#reference.variants.descriptions), otherwise the variant block will be interpreted as a text description, I guess? Why was this keyword added?

I thought the same thing so tested it but it looks like the keyword by itself is ignored and the default description is used.  Since there are default global descriptions for +x11 and +quartz there's really no reason to add a description here.

Reply:
Anyway, I removed the unnecessary `description` keyword.

Reply:
Oh, interesting, I didn't know that, and it seems undocumented..?